scale/docs/rest/v6/data.rst

```diff
@@ -74,12 +74,73 @@ An interface JSON describes a set of parameters that can be passed data values t
 |                            |                |          | 'array', 'boolean', 'integer', 'number', 'object', or 'string'.    |
 +----------------------------+----------------+----------+--------------------------------------------------------------------+
 
+
 .. _rest_v6_data_filter:
 
 Data Filter JSON
 ----------------
 
-A data filter JSON describes a set of filters that determines whether a set of data is accepted or not.
+A data filter JSON describes a set of filters that determines whether a set of data is accepted or not.  A filter consists of a name, type,
+condition and a set of values.
+
+The name determines which input parameter is examined. The value 'input_a' means the filter will look at the value of that parameter if it exists.
```

Comment:
"The name determines which *condition* input parameter is examined"? Is there a way to clarify where the input parameter comes from?